### PR TITLE
[#7870] Improve documentation for atomic acls/metadata microservices (main)

### PR DESCRIPTION
**Pull request by korydraughn**
Example code works as intended.

### CODE REVIEW

File: plugins/microservices/src/atomic_apply_acl_operations.cpp [hunk 1]
```diff
@@ -126,6 +127,26 @@ auto plugin_factory() -> irods::ms_table_entry*
 /// \retval 0        On success.
 /// \retval non-zero On failure.
 ///
+/// \b Example
+/// \code{.py}
+/// # The collection to apply the ACL operations to.
+/// *lpath = '/tempZone/home/alice';
```
Comment (trel) [inline plugins/microservices/src/atomic_apply_acl_operations.cpp:133]: does a trailing slash matter here? is it worth noting if it DOES matter?

Comment (korydraughn) [reply plugins/microservices/src/atomic_apply_acl_operations.cpp:133]: It's not supposed to matter, but I bet the current implementation of the API doesn't handle it. Will give it a try.

I don't think it's worth mentioning. The user shouldn't need to worry about trailing slashes. We have a function which takes care of that.

Comment (korydraughn) [reply plugins/microservices/src/atomic_apply_acl_operations.cpp:133]: Confirmed the API doesn't handle trailing slashes.

Will write up and fix as part of this PR.

Comment (korydraughn) [reply plugins/microservices/src/atomic_apply_acl_operations.cpp:133]: Well, I'll write an issue.

Comment (korydraughn) [reply plugins/microservices/src/atomic_apply_acl_operations.cpp:133]: Created #7892.

Resolving since that is out-of-scope of this PR.

**Comment by korydraughn** [comment]
#'d

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**